avaje-config/src/main/java/io/avaje/config/Parsers.java

```diff
@@ -35,11 +35,16 @@ private void initYamlParser() {
   }
 
   private void initParsers() {
-    ServiceLoader.load(ConfigParser.class).forEach(p -> {
-      for (var ext : p.supportedExtensions()) {
-        parserMap.put(ext, p);
-      }
-    });
+    ServiceLoader.load(ConfigSPI.class)
```

Comment:
Hmm. I was expecting this code to live with all ServiceLoader code in a specific class dedicated to that job. Pondering ...

Reply:
Service Loader is memoized so I thought was fine